src/components/PathwayCourse/index.js

```diff
@@ -132,12 +131,16 @@ function PathwayCourse() {
   const [openSnackbar, setOpenSnackbar] = useState(false);
   const [openModal, setOpenModal] = useState(false);
   const [certificate, setCertificate] = useState("");
-  let completedAll = pathwayId==8?completedPortion?.total>=80: completedPortion?.total === 100;
+  let completedAll =
```

Comment:
`completedAll` is a little misleading as a variable name if you only need to complete a minimum percentage. Also, use `const` here since the variable is not reassigned.